util/src/main/java/io/kubernetes/client/util/leaderelection/Lock.java

```diff
@@ -0,0 +1,16 @@
+package io.kubernetes.client.util.leaderelection;
+
+public interface Lock {
+
+  String LeaderElectionRecordAnnotationKey = "control-plane.alpha.kubernetes.io/leader";
```

Comment:
the annotation key should not be defined as a part of the interface, the underlying implementation of resource lock in k/l is migrating to `coordination.Lease` which provides corresponding api fields in the spec instead of annotation hacks.